tests/ops/functions/test_equal.py

```diff
@@ -1702,6 +1702,27 @@ def test_prod_with_multi_wire_bases(self, base_list1, base_list2, res):
         op2 = qml.prod(*base_list2)
         assert qml.equal(op1, op2) == res
 
+    def test_prod_of_prods(self):
+        """Test that prod of prods and just an equivalent Prod get compared correctly"""
+        X = qml.PauliX
+        qml.operation.enable_new_opmath()
+        op1 = (
+            0.5
+            * X(0)
```

Comment:
This formatting seems weird. I guess it should be fine if `black` isn't complaining.

Reply:
This _was_ actually black's doing. But I agree it looks odd so I updated the test to single line